ingestion/src/metadata/ingestion/source/database/dbt/dbt_service.py

```diff
@@ -225,6 +206,9 @@ def get_dbt_objects(self) -> Iterable[DbtObjects]:
         self.remove_manifest_non_required_keys(
             manifest_dict=self.context.get().dbt_file.dbt_manifest
         )
+        self.remove_run_result_non_required_keys(
+            run_results=self.context.get().dbt_file.dbt_run_results
```

Comment:
Is it possible to end up with a None value when calling `self.context.get()`? If so we should handle it so we don't end up with some `AttribiuteErrro`

Reply:
Added the check there